scalafmt-tests/src/test/resources/test/StripMargin.stat

```diff
@@ -1,15 +1,62 @@
 assumeStandardLibraryStripMargin = true
 <<< Align | margin 1
 val x = """Formatter changed AST
-          |=====================
-          |$diff
+      |=====================
```

Comment:
This looks like a regression. The assuneStandardLibraryStripMargin setting aligns multilingual strings by default and we should try to keep that behavior.

Reply:
The alignment is disabled by default since the assumption can produce semantic changes.

Reply:
@olafurpg this is the *input* to the test (it was already aligned, so i decided to change it to emphasize the result). the output is unchanged. apologies for the confusion.

the only change to default is, `align.stripMargin` is set to false when `align = none` (seemed a natural thing to do). everything else is the same.

Reply:
> The alignment is disabled by default since the assumption can produce semantic changes.

can you please share an example?

Reply:
My bad 😅 

> The alignment is disabled by default since the assumption can produce semantic changes.

Users can implement their own `stripMargin` method with custom semantics where the indentation  spaces have significant meaning. A contrived example
```scala
implicit class XtensionStripMargin(string: String) {
  def stripMargin = string.length
}
"""a
  |b
  |""".stripMargin
```

What is the difference between these two configs?

```
assumeStandardLibraryStripMargin = true
align.stripMargin = true
```

and 
```
assumeStandardLibraryStripMargin = true
```

Reply:
> What is the difference between these two configs?
> 
> ```
> assumeStandardLibraryStripMargin = true
> align.stripMargin = true
> ```
> 
> and
> 
> ```
> assumeStandardLibraryStripMargin = true
> ```

`assumeStandardLibraryStripMargin` allows the rewrite of indentation within the string constant (if it's false, nothing is modified); `align.stripMargin` defines the indentation (aligned or not).

```
// align.stripMargin = true
val a = s"""foo
           | bar""".stripMargin

// align.stripMargin = false
val a = s"""foo
  | bar""".stripMargin
```
it looks like a number of people like some sort of alignment, but we do not. in any form.

Reply:
I see, thanks for the explanation. It would help if the PR description of changes like these included more details. For example

---
Previously, when using `assumeStandardLibraryStripMargin = true` it was not possible to enable vertical alignment of `|` characters in multiline `.stripMargin` strings using 2-space indentation. The `|` characters always aligned by the opening triplequote `"""`. Now, users can optionally enable `align.stripMargin = false` to align the `|` character at 2-spaces instead of by the opening triplequote.

```scala
// With align.stripMargin = true
val x = """
        |Hello world!
        |""".stripMargin
// With align.stripMargin = false
val x = """
  |Hello world!
  |""".stripMargin
```

Reply:
will do!